src/go/rpk/.goreleaser.yaml

```diff
@@ -0,0 +1,71 @@
+project_name: rpk
+builds:
+  - id: rpk-windows
```

Comment:
This is only for windows & darwin, are we expecting linux builds to default to the full redpanda package?

Reply:
It looks like the old code had a release for ubuntu directly, not sure where linux is in the new version.

Reply:
this is just for binaries released via github releases. the deb/rpm/.tgz packages for linux that install both redpanda/rpk are not affected by this PR. see related https://github.com/redpanda-data/redpanda/issues/4191#issuecomment-1305182327

Reply:
Interesting. Maybe we can publish linux-specific rpk-only packages in the future, but that can be a separate issue.